x-pack/plugins/security_solution/server/endpoint/routes/actions/isolation.ts

```diff
@@ -5,81 +5,146 @@
  * 2.0.
  */
 
-import { schema } from '@kbn/config-schema';
+import { schema, TypeOf } from '@kbn/config-schema';
+import moment from 'moment';
 
-import { IRouter } from 'src/core/server';
+import { RequestHandler } from 'src/core/server';
+import uuid from 'uuid';
+import { EndpointAction } from '../../../../common/endpoint/types';
+import { ISOLATE_HOST_ROUTE, UNISOLATE_HOST_ROUTE } from '.';
+import {
+  SecuritySolutionPluginRouter,
+  SecuritySolutionRequestHandlerContext,
+} from '../../../types';
+import { GetAgentIDsForEndpoints } from '../../services';
 import { EndpointAppContext } from '../../types';
 
+export const HostIsolationRequestSchema = {
```

Comment:
Normally we want the UI and Server to both use the same types. I would suggest moving the schema to `x-pack/plugins/security_solution/common/endpoint/schema/` and also creating `TypeOf<>` for them there or under `x-pack/plugins/security_solution/common/endpoint/types`